batavia/core/builtins.js

```diff
@@ -35,6 +35,23 @@ batavia.builtins.__import__ = function(args, kwargs) {
     if (module === undefined) {
         module = batavia.modules.sys.modules[args[0]];
     }
+    // Check if there is a stdlib (pyc) module.
+    if (module === undefined) {
```

Comment:
using `module === undefined` twice hear makes me read it as "if this is *still* undefined, then try this other thing. Is that accurate?